health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundariesDuplicate.js

```
@@ -0,0 +1,153 @@
+            <Wrapper
+              hierarchyType={hierarchyType}
+              lowest={lowestHierarchy}
+              selectedData={selectedData}
+              boundaryOptions={boundaryOptions}
+              hierarchyData={props?.props?.hierarchyData}
+              isMultiSelect={"true"}
+              restrictSelection = {restrictSelection}
+              onSelect={(value) => {
+                handleBoundaryChange(value);
+              }}
+            ></Wrapper>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)